projects/plugins/jetpack/extensions/blocks/contact-form/child-blocks.js

```diff
@@ -546,13 +548,112 @@ export const childBlocks = [
 			icon: renderMaterialIcon(
 				<Path fill={ getIconColor() } d="M3 17h18v2H3zm16-5v1H5v-1h14m2-2H3v5h18v-5zM3 6h18v2H3z" />
 			),
-			edit: editMultiField( 'select' ),
+			edit: ( { attributes, isSelected, setAttributes } ) => {
```

Comment:
This piece isn't working perfectly yet.
Need to adjust some behavior and move it out to a separate file.